### PR TITLE
Closes #1669

### LINKED ISSUE #1669: TimeoutException after extended clustered get/put test
I’m running a single-client 90/10 get/put test with a 16gb L2 cache and no L1 cache with 128 test threads.  After approx. 60-90 minutes we start getting TimeoutExceptions on the client, after which the client hangs – attempts to manually restart the client are unsuccessful.  The client and server are both using min/max 2gb heaps.

### PR DESCRIPTION
**Pull request by AbfrmBlr**

### CODE REVIEW

**Comment by AbfrmBlr** [comment]
This PR fixes the problem reported.

**Comment by kcleereman** [comment]
There is no performance drop in "get" latency caused by this PR

File: clustered/server/src/main/java/org/ehcache/clustered/server/offheap/OffHeapChainMap.java [hunk 1]
```diff
@@ -48,31 +48,34 @@
...
-  private volatile ChainMapEvictionListener<K> evictionListener;;
+  private volatile ChainMapEvictionListener<K> evictionListener;
 
   public OffHeapChainMap(PageSource source, Portability<? super K> keyPortability, int minPageSize, int maxPageSize, boolean shareByThieving) {
-    this.chainStorage = new OffHeapChainStorageEngine<K>(source, keyPortability, minPageSize, maxPageSize, shareByThieving, shareByThieving);
-    EvictionListener<K, InternalChain> listener = new EvictionListener<K, InternalChain>() {
-      @Override
-      public void evicting(Callable<Map.Entry<K, InternalChain>> callable) {
+    this.chainStorage = new OffHeapChainStorageEngine<>(source, keyPortability, minPageSize, maxPageSize, shareByThieving, shareByThieving);
+    EvictionListener<K, InternalChain> listener = callable -> {
```
Comment (chrisdennis) [inline clustered/server/src/main/java/org/ehcache/clustered/server/offheap/OffHeapChainMap.java]: I know this code can be JDK 1.8, but it seems a bit needless to change it now.  Just obscures the rest of the changes.

Comment (AbfrmBlr) [inline clustered/server/src/main/java/org/ehcache/clustered/server/offheap/OffHeapChainMap.java]: Done

**Comment by AbfrmBlr** [comment]
test this please

**Pull request merged**